proprietary/tokens/src/components/ams/unordered-list.tokens.json

```
@@ -21,6 +21,7 @@
       },
       "unordered-list": {
         "list-style-type": { "value": "'\\2013'" },
+        "padding-block-start": { "value": "{ams.space.xs}" },
```

The designers do think `sm` is better here.

```suggestion
        "padding-block-start": { "value": "{ams.space.sm}" },
```